packages/frontend/src/state/positions/providers.tsx

```diff
@@ -98,9 +99,20 @@ export const ComputeSwapsProvider: FC = ({ children }) => {
   useAppEffect(() => {
     if (computedSwaps.squeethAmount.isGreaterThan(0) && oSqueethBal?.isGreaterThan(0)) {
       setPositionType(PositionType.LONG)
+      // check if user osqth wallet balance is equal to the accumulated amount from tx history
+      // if it's not the same, it's likely that they do smt on crab acution or otc or lp etc so dont show the pnl for them
+      if (!computedSwaps.squeethAmount.isEqualTo(oSqueethBal)) {
```

Comment:
is there any chance that a user who only has a long position may have their squeethAmount not be equal to the osqueethBal?